eng/pipelines/libraries/run-test-job.yml

```diff
@@ -71,6 +72,9 @@ jobs:
             artifactName: $(librariesTestsArtifactName)
             artifactFileName: $(librariesTestsArtifactName)$(archiveExtension)
 
+        - script: $(_buildScript) -restore $(_buildArguments) $(_skipTestRestoreArg)
```

Comment:
Please don't do a full restore. Why do you need to run restore? Is this to be able to run `pretest.proj`?

Reply:
i believe it needs installer.tasks to be restored at least.

Reply:
Why is that needed? Running restore adds like 2 minutes to the build. So I want to understand why and see if we can add a workaround instead.

Reply:
because GenerateTestSharedFrameworkDepsFile task in pretest.proj is implemented there.

Reply:
I see. We should be able to change  the command to do:

`$(_buildScript) -project src/libraries/pretest.proj ....`

That will ensure to use the root build script and will restore and build the installer tasks, but it will not do a full restore of the libraries subset.

Reply:
ok trying $(_buildScript) -project**s** src/libraries/pretest.proj now, also a fix to proj.test as suggested on chat.